google-cloud-storage/conformance/v1/v4_signatures.json

```diff
@@ -288,10 +288,10 @@
           "x-goog-algorithm": "GOOG4-RSA-SHA256",
           "x-goog-credential": "test-iam-credentials@dummy-project-id.iam.gserviceaccount.com/20200123/auto/storage/goog4_request",
           "x-goog-date": "20200123T043530Z",
-          "x-goog-signature": "925766caea79216580bff38d89b14f7f5b45719f54a87224b39b21592f60cad3aa1ec42b3e39b01c8a14420133a029fe5f32b4e925b1acd9c5e24aadaea70f9dddeb75dd14cf5ca7464e5bcffac576d2645f558f086e0c9b6ba67bdc0cc3e7543b1aa08aa491549a6f6d7b303b39751cab7edb5222f5f9a11a961442030475fc1485df631a97c81958b252c37dbcf2bb2bfb6ecd589d46a0fe4298f4f1f0c5b23b1b8ae57c0f522a786d508c5311270552f27b8940917ba3f1b8004e1f4c80340de3de859ebd795ee7ca5bf382c29c5fa9e47f2de915fdc0dcfdb764dca33943062316bb3fe4c13615c10cfb107daa92fa49187552c918c6fd6aa3a272f0d9fb",
```

Comment:
@quartzmo general question, did you find the conformance test structure for POST policies to be confusing?

Reply:
No, it makes sense to me.